src/main/java/org/folio/list/domain/ListEntity.java

```diff
@@ -18,12 +18,15 @@
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
+import lombok.With;
+
 import org.folio.list.domain.dto.ListUpdateRequestDTO;
 import org.folio.list.exception.AbstractListException;
 import org.folio.list.rest.UsersClient.User;
 import org.folio.list.util.TaskTimer;
 
 @Data
+@With
```

Comment:
[This](https://projectlombok.org/features/With) generates methods like `withIsDeleted`, `withName`, etc., which clones and edits one field in a fluent manner.

Reply:
Totally out of scope and I'm not asking for any changes, but I just need to complain...

ngl, I really hate everything about this class, but I don't care enough to go out of my way to change it... It's like a perfect storm of things that annoy me: lombok + ORM + mutability + leaky abstraction (it has a dependency on something in `UserClient`) + business logic in a data class (with no comments, no less)

Reply:
Agreed. We could do a quick clean up noise-wise at least by removing the `@Column`'s (it'll derive from fields), and changing some to primitives (removes @NotNull's) — super minor change, but would save a smidge of sanity